TrackingTools/TrackRefitter/interface/TrackTransformer.h

```diff
@@ -44,6 +44,9 @@ class TrackTransformer final : public TrackTransformerBase{
 
   /// Destructor
   ~TrackTransformer() override;
+
+  /// fillDescriptions
+  static void fillDescriptions(edm::ParameterSetDescription& descriptions);
```

Comment:
I'd suggest to rename this as `fillPSetDescription()`, because
* the function does not fill a description of a module (but a PSet)
* the function fills a description of only a single PSet
* such a convention is already been used elsewhere (especially in the mechanism for plugin factories)